### PR TITLE
ForgeRock SDK + PingOne #186988679

### PR DESCRIPTION
**Pull request by jon-oblander**

### CODE REVIEW

File: fr_sdk-reg-authn-pkg/sample-app/dashboard.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        redirectUri: `https://localhost:8080/dashboard.html`, 
+        scope: 'openid profile email revoke', 
+        serverConfig: {
+          baseUrl: `https://auth.pingone.${window._env_.pingOneDomain}`,
+          paths: {
+            authorize: `/${window._env_.pingOneEnvId}/as/authorize`,
+            accessToken: `/${window._env_.pingOneEnvId}/as/token`,
+            revoke: `/${window._env_.pingOneEnvId}/as/revoke`,
+            userInfo: `/${window._env_.pingOneEnvId}/as/userinfo`,
+            endSession: `/${window._env_.pingOneEnvId}/as/signoff`, // Managing session varies across products which may result in JavaScript errors during log out.
```
Comment (michaelsanchez-pingidentity) [inline fr_sdk-reg-authn-pkg/sample-app/dashboard.html:62]: Let's just mod this a bit to, "The endpoints for managing sessions vary across products, and may result in seeing JavaScript errors in the developer console during the logout process."

**Review by michaelsanchez-pingidentity — CHANGES_REQUESTED** [review]
Just one comment inline with some text modifications.

File: fr_sdk-reg-authn-pkg/sample-app/dashboard.html (hunk 1 above)
Comment (jon-oblander) [reply fr_sdk-reg-authn-pkg/sample-app/dashboard.html]: updated and pushed.

**Review by michaelsanchez-pingidentity — APPROVED** [review]
Reviewed.

**Pull request merged**